.github/actions/time-compilation/action.yaml

```diff
@@ -0,0 +1,43 @@
+name: Time Compilation
+description: Time the cargo compilation, outputting an HTML file.
+
+inputs:
+  target:
+    description: "Target triple, like `wasm32-unknown-unknown"
+    required: false
+  target_option:
+    description: "--target={target}"
+    required: false
+  use_cache:
+    required: true
+    type: boolean
+
+runs:
+  using: composite
+  steps:
+    - name: 🔗 Setup toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        target: ${{ inputs.target }}
+        toolchain: nightly
+        override: true
+        default: true
+    - name: 💰 Cache
+      id: cache
+      uses: Swatinem/rust-cache@v1
+      # 'true' seems to require quotes (and using `inputs.use_cache` doesn't
+      # work); I'm really not sure why. There are some issues on the interwebs
+      # around this, but I couldn't find one that explained it.
+      if: inputs.use_cache == 'true'
+    - name: 🏭 Compile
+      uses: actions-rs/cargo@v1
+      with:
+        command: build
+        args: --timings=html -Z unstable-options
```

Comment:
timings is now stable, no need for nightly or -Z

Reply:
Thanks. IIUC `--timings=html` is not stable though... (arguably we shouldn't be using that for CI — it does make things easier to look at though!)